### PR TITLE
 adding directionality count

### PR DESCRIPTION
**Pull request by LiyaoJiang1998**
Adding directionality count to the csv mode

### CODE REVIEW

File: src/thermal_save_csv.py [hunk 1]
```diff
@@ -208,10 +206,10 @@ def main():
                                      (displayPixelHeight * ix, displayPixelWidth * jx, displayPixelHeight, displayPixelWidth))
                 except:
                     print("Caught drawing error")
-        pygame.display.update()
+        #pygame.display.update()
```
Comment (kylecjo) [inline src/thermal_save_csv.py:209]: I commented this out when testing, you can delete it completely

**Comment by LiyaoJiang1998** [comment]
- using class TrackableObject to detect the direction of objects
- add directional count to the payload, and save to csv file
- display the line of crossing and directional count in pygame

File: src/thermal_save_csv.py [hunk 2]
```diff
@@ -261,23 +265,29 @@ def main():
...
                         to.counted = True
 
             # store the trackable object in our dictionary
             trackableObjects[objectID] = to
 
         # update counter in top left
         # textsurface = myfont.render(str(ct.get_count()), False, (0, 0, 0))
-        textsurface = myfont.render(str(totalUp)+' : '+str(totalDown), False, (0, 0, 0))
-        lcd.blit(textsurface,(0,0))
+        # if total_up != total_up_old or total_down != total_down_old:
```
Comment (kylecjo) [inline src/thermal_save_csv.py:283]: You can delete this comment and the one directly above

**Review by kylecjo — CHANGES_REQUESTED** [review]
Looks good, I added a few notes on which comments should be removed. The commits are kinda wonky so we should try to fix those before merging.

**Comment by kylecjo** [comment]
Closed with #29

**Pull request closed without merging**